migrations/versions/ec7604eaad1a_.py

```diff
@@ -0,0 +1,54 @@
+"""empty message
```

Comment:
Remember to add a message to your migrations with `-m` when running the migration step.